src/app/base/components/MainContentSection/MainContentSection.tsx

```diff
@@ -36,11 +40,13 @@ const MainContentSection = ({
             </Col>
           )}
           <Col size={(sidebar ? TOTAL - SIDEBAR : TOTAL) as ColSize}>
-            {!isNotificationListHidden && <NotificationList />}
-            {children}
+            <ContentSection.Content>
```

Comment:
This means that a potential footer will be nested within content. This shouldn't break anything, but we don't have such an example in the component library. Could you please add it, then this should be good to go.